fhir-bulkimportexport-webapp/src/main/java/com/ibm/fhir/bulkimport/ImportJobListener.java

```diff
@@ -49,26 +66,25 @@ public void afterJob() {
             }
         }
 
-        jobEndTimeInMS = System.currentTimeMillis();
-        double jobProcessingSeconds = (jobEndTimeInMS - jobStartTimeInMS)/1000.0;
+
+        double jobProcessingSeconds = (totalJobExecutionMilliSeconds)/1000.0;
         jobProcessingSeconds = jobProcessingSeconds < 1 ? 1.0 : jobProcessingSeconds;
 
-        // Print out the simple metrics to console.
-        System.out.println(" ---- Fhir resources imported in " + jobProcessingSeconds + "seconds ----");
-        System.out.println("ResourceType \t Imported \t Failed");
+        // log the simple metrics.
+        logger.info(" ---- Fhir resources imported in " + jobProcessingSeconds + "seconds ----");
+        logger.info("ResourceType \t Imported \t Failed");
```

Comment:
is this outputting a report?

Reply:
yes, this is the report that I pasted into the issue for tracking progress...